src/config/env.ts

```diff
@@ -0,0 +1,40 @@
+export const ENV = {
+  PRODUCTION: 'production',
+  DEVELOPMENT: 'development',
+  TEST: 'test',
+};
```

Comment:
Should we add this to sdk too 🤔

Reply:
With vite we can just check for `import.meta.env.DEV` where we do `NODE_ENV === ENV.DEVELOPMENT`. So we should be able to remove the `ENV`. And use the `mode` from vite instead.